tests/python/driver/tvmc/test_compiler.py

```diff
@@ -87,6 +88,16 @@ def test_compile_tflite_module(use_vm, tflite_mobilenet_v1_1_quant):
     verify_compile_tflite_module(tflite_mobilenet_v1_1_quant, shape_dict, use_vm=use_vm)
 
 
+def test_tir_dump(tflite_mobilenet_v1_1_quant):
+    pytest.importorskip("tflite")
+    tvmc_model = tvmc.load(tflite_mobilenet_v1_1_quant)
+    tvmc_package = tvmc.compile(tvmc_model, target="llvm", dump_code="tir")
+    dumps_path = tvmc_package.package_path + ".tir"
+    assert os.path.exists(dumps_path)
+    with open(dumps_path) as f:
+        assert "tir" in f.read()
```

Comment:
@lhutton1, feels great to see this addition to `dump_code` :smile: Just a small question about testing: does this file already have a test for dumping multiple IRs from the same command?

Reply:
It does not unfortunately, I'll add one

Reply:
Added